e2e/js/js-esm/README.md

```diff
@@ -0,0 +1,24 @@
+# DevCycle JS SDK CDN E2E Test
+
+## Getting started
+
+If you want to run the e2e app, from the root folder, you'll have to build the JS SDK first:
+
+```bash
+yarn nx build js
+```
+
+Then you can run the e2e app from the root folder:
+
+```bash
+yarn e2e:js-esm:start
```

Comment:
running this should just automatically build the js sdk

Reply:
it already does, i'll update the readme